dev/docker/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+docker run -it --rm\
+    -v $(cd ../../; pwd):/root/workspace/Chaos\
+    -p 8082:80\
+    -p 8081:8081\
+    chaos\
+    python -m unittest
```

Comment:
@phil-r is right, I just tested! (Otherwise it makes a mistake)
And it's `python -m unittest discover -p *_test.py` without the `"` in @phil-r's answer :)